### ISSUE TITLE
Enhancement - Grid digitizer - Allow upload of identifiers with non-numerical characters

### ISSUE THREAD

**Issue by tmcelrath** [SpeciesFileGroup/taxonworks#1545]
When I am in the grid digitizer, and I want to upload an identifier with non numerical characters E.g. "G12345" then I cannot because that field does not allow for non-numerical characters. I'm assuming that this will mess with the auto-generated identifiers function. 

However, many collections use non-numerical characters that are partially numerical. We need to add this functionality, and the INHS insect collection needs it asap to upload "S123456" numbers. 

Can we improve the identifier auto-generator to allow for partially numeric identifiers?

**Comment by tmcelrath** [comment]
@mjy Thoughts?

**Comment by tmcelrath** [comment]
@jlpereira thoughts?

**Comment by jlpereira** [comment]
@tmcelrath right, we're accepting only numbers. I think I could make some changes on increment code at the interface. But not sure if the backend already handle this, probably not. We need to tackle this on both sides

**Comment by kleintom** [comment]
Works for me:

![Image](https://github.com/user-attachments/assets/677e9ea4-928c-4c5b-8abb-faa48c8d8315)

**Comment by tmcelrath** [comment]
Was this changed on the backend? I know it works in comprehensive, but glad to know it works on Grid now.
